tests/dragonfly/utility.py

```diff
@@ -588,12 +603,18 @@ async def _executor_task(self, db, queue):
                 queue.task_done()
                 break
 
-            pipe = client.pipeline(transaction=tx_data[1])
-            for cmd in tx_data[0]:
-                pipe.execute_command(*cmd)
-
             try:
-                await pipe.execute()
+                if self.fake_redis is None:
+                    pipe = client.pipeline(transaction=tx_data[1])
+                    for cmd in tx_data[0]:
+                        pipe.execute_command(*cmd)
+                    await pipe.execute()
+                else:
+                    # To mirror consistently to Fake Redis we must only send to it successful
```

Comment:
this is not so good that we dont test multi flow but its better than nothing..
I will continue thinking if we have a better approach to testing correctness of migraion so we will not have this limitaion